Noura_Alsubaie/week_01/day_02/HomeWork/variabelsTask.js

```diff
@@ -0,0 +1,21 @@
+// Fortune Teller
+function fortune() {
+var numOfChildren = [1, 4, 3];
+var  partnerName = [null, null, null];
+var geographicLocation= [null, null, null];
+var jobTitle = [null, null, null];
+var randomChildren = Math.floor(Math.random() * (numOfChildren.length));
+    console.log("you will have  " + randomChildren + "children ");
+}
+fortune;
```

Comment:
So this will print a random number of children, but how about the rest? How can we print them?